memcall_solaris.go

```diff
@@ -0,0 +1,91 @@
+package memcall
```

Comment:
You'll want build constraints here to only build this file on solaris.

Reply:
Thank you for looking at it, and no problem. It should not be necessary when you specify `_solaris` in the name. This code will _only_ build on Solaris and illumos. If you insist that it be added, I can certainly do it.

Thanks again.

Reply:
Changes made as per your recommendation. Let me know if you have any other thoughts.

Here are results of building on OmniOS, which behaves at least in this regard identical to Solaris.
```
SunOS szaydel-omnios-04 5.11 omnios-r151050-49db1c0a0fe i86pc i386 i86pc
omni@szaydel-omnios-04:~/memcall$ git checkout bug/szaydel/illumos-support
branch 'bug/szaydel/illumos-support' set up to track 'origin/bug/szaydel/illumos-support'.
Switched to a new branch 'bug/szaydel/illumos-support'
omni@szaydel-omnios-04:~/memcall$ go build .
omni@szaydel-omnios-04:~/memcall$ echo $?
0
omni@szaydel-omnios-04:~/memcall$ git checkout master
Switched to branch 'master'
Your branch is up to date with 'origin/master'.
omni@szaydel-omnios-04:~/memcall$ go build .
# github.com/awnumar/memcall
./memcall_unix.go:15:23: undefined: unix.MADV_DONTDUMP
```

Reply:
Is OmniOS supported by Go?

Reply:
It is supported in so far as illumos is supported and there are actually specific carveouts for illumos.